Sources/FloatingButton/FloatingButton.swift

```diff
@@ -12,6 +12,10 @@ public enum Direction {
     case left, right, top, bottom
 }
 
+public enum Rotation {
+    case clockwise, counterclockwise
```

Comment:
could you please use camel case - counterClockwise

Reply:
BTW, I *really* like this project, it is extremely well done and I find it very useful!